service/azureconfig/v1/arm_templates/peering.json

```diff
@@ -0,0 +1,41 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "name": {
+      "type": "string"
+    },
+    "sourceNetwork": {
+      "type": "string"
+    },
+    "targetNetwork": {
+      "type": "string"
+    },
+    "resourceGroup": {
+      "type": "string"
+    },
+    "apiVerison": {
```

Comment:
apiVersion?

Reply:
actually I renamed this to `virtualNetworksPeeringAPIVersion` and provided a default value and description